src/python/WMCore/WMRuntime/DashboardInterface.py

```diff
@@ -257,34 +262,34 @@ def stepStart(self, step):
 
         data = None
         if not self.jobStarted:
-            #It's the first step so let's send the exe that started and where
-            #That's what they request
+            # It's the first step so let's send the exe that started and where
+            # That's what they request
             data = {}
-            data['MessageType']       = 'jobRuntime-jobStart'
-            data['MessageTS']         = time.strftime(self.tsFormat,
-                                                      time.gmtime())
-            data['taskId']            = self.taskName
-            data['jobId']             = self.jobName
-            data['GridName']          = getUserProxyDN()
-            data['ExeStart']          = helper.name()
-            data['SyncCE']            = getSyncCE()
-            data['WNHostName']        = socket.gethostname()
-
-            self.publish(data = data)
+            data['MessageType'] = 'jobRuntime-jobStart'
+            data['MessageTS'] = time.strftime(self.tsFormat,
+                                              time.gmtime())
+            data['taskId'] = self.taskName
+            data['jobId'] = self.jobName
+            data['GridName'] = getUserProxyDN()
+            data['ExeStart'] = helper.name()
+            data['SyncCE'] = getSyncCE()
+            data['WNHostName'] = socket.gethostname()
+
+            self.publish(data=data)
             self.jobStarted = True
 
-        #Now let's send the step information
+        # Now let's send the step information
         tmp = {'jobStart': data}
 
         data = {}
         data['MessageType'] = 'jobRuntime-stepStart'
-        data['MessageTS']   = time.strftime(self.tsFormat, time.gmtime())
-        data['taskId']      = self.taskName
-        data['jobId']       = self.jobName
-        data['%d_stepName' % self.stepCount]    = helper.name()
-        data['%d_ExeStart' % self.stepCount]    = helper.name()
+        data['MessageTS'] = time.strftime(self.tsFormat, time.gmtime())
+        data['taskId'] = self.taskName
+        data['jobId'] = self.jobName
+        data['%d_stepName' % self.stepCount] = helper.name()
+        data['%d_ExeStart' % self.stepCount] = helper.name()
```

Comment:
You are right. Although it is weird that we need two different key, I guess this is what was expected. Thanks for checking into